src/DownloadFormat/DownloadFormat.styles.ts

```diff
@@ -0,0 +1,74 @@
+import styled, { css } from 'styled-components';
+
+interface OptionProps {
+  variant?: string;
+}
+
+export const FormatOption = styled.div<OptionProps>`
+  text-align: center;
+  font-weight: bold;
+  height: 36px;
+  font-size: 12px;
+  line-height: 16px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &:first-child {
+    border-radius: 5px 5px 0 0;
+  }
+
+  &:last-child {
+    border-radius: 0 0 5px 5px;
+  }
+
+  &:hover {
+    background-color: ${p => p.theme.colors.lightSkyBlue};
+    color: ${p => p.theme.colors.white};
+  }
+
+  &:active {
+    background-color: ${p => p.theme.colors.darkSkyBlue};
+  }
+
+  &:not(:last-child) {
```

Comment:
Whoa, :not has decent coverage, I didn't know that - https://caniuse.com/#search=%3Anot